src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java

```diff
@@ -1445,22 +1450,15 @@ public void run() {
         "BacklogIssuePanel.message.uploading.attachments=Uploading files"
     })
     private void selectFiles() {
-        // attachment
-        RP.post(new Runnable() {
 
+        class AttachmentUploader implements Runnable {
```

Comment:
I thought that local class would be similar to the anonymous one, and was created because i was unable to put `File[] attachments` in `selectFiles()` scope and use it from the anonymous `Runnable.run()` inner scope.
I like the private static nested class, but static would require a way to share current IssuePanel instance properties?

Anyway now I see that a `final File[] attachments` in `selectFiles()` scope will allow the access from the anonymous Runnable passed to RP. So maybe the change doesn't need to declare a class and just extract `File[] attachments = FileChoseBuilder ... ` outside the `Runnable.run()`. 
( I need to pay more attention to the clever netbeans! )

I am somewhat embarrassed because I feel lost between my lack of knowledge and OOP subjectivity.

Probably you could address this fix better, and because I'm not sure how to now revert this commits (Revert the pushed commits can affect Github badly, or leave an ugly history right?).
Thanks.